server/controllers/refer/courseParticipationsController.ts

```diff
@@ -149,8 +149,13 @@ export default class CourseParticipationsController {
         courseParticipations,
         req.user.token,
       )
-      const summaryListsOptions = courseParticipationsWithNames.map(courseParticipationWithName =>
-        CourseParticipationUtils.summaryListOptions(courseParticipationWithName, referral.id),
+      const sortedCourseParticipationsWithNames = courseParticipationsWithNames.sort(
```

Comment:
Alternatively could use `localeCompare` e.g.

```
const sortedCourseParticipationsWithNames = courseParticipationsWithNames.sort((participationA, particiationB) => participationA.createdAt.localeCompare(particiationB.createdAt))
```

Reply:
Yeah I wondered about that since we use it elsewhere, but wasn't sure if it made a difference with date strings. I suppose it's more concise in any case